gradio/inputs.py

```diff
@@ -19,12 +19,39 @@
 from ffmpy import FFmpeg
 
 from gradio import processing_utils, test_data
-from gradio.component import Component
+from gradio.component import Component, Textbox
```

Comment:
Suggest importing components as:

`from gradio import components`

And then having something like:

`class Textbox(components.Textbox)` for readability?

Reply:
Well actually importing the related class and using it directly is corresponding with PEP-8 as far as I know :)

Reply:
Oh I see